Source/MQTTnet/Server/MqttSubscriptionInterceptorContext.cs

```diff
@@ -20,8 +22,37 @@ public MqttSubscriptionInterceptorContext(string clientId, TopicFilter topicFilt
         /// </summary>
         public IDictionary<object, object> SessionItems { get; }
 
-        public bool AcceptSubscription { get; set; } = true;
+        private bool acceptSubscription = true;
+        public bool AcceptSubscription
+        {
+            get => acceptSubscription;
+            set {
+                if (!value && resultCode < MqttSubscribeReasonCode.UnspecifiedError) resultCode = MqttSubscribeReasonCode.UnspecifiedError;
```

Comment:
Please stick to the code style used in this project. There is nothing bad in your implementation but we just use curly braces in every case. So please adopt the style here as well. And below also 😄